testing/sigrok-cli/APKBUILD

```
@@ -1,20 +1,16 @@
 # Maintainer: Marian Buschsieweke <marian.buschsieweke@ovgu.de>
-
-pkgname="sigrok-cli"
+pkgname=sigrok-cli
```

```suggestion
pkgname="sigrok-cli"
```

I'm never going to approve unquoting values for variable assignments. This is a step backward in my opinion.